lessondance.tex

```diff
@@ -0,0 +1,109 @@
+\documentclass{lessonplan}
+
+% Constants
+\newcommand{\lessonNumber}{Dance}
+\title{Dance Dance Revolution}
+\author{\linkHome}
+\date{}
+
+% Buildup
+\setLessonNumber{\lessonNumber}
+
+
+%Document
+\begin{document}
+
+  \maketitle
+
+  \section{Overview}
+    \subsection{Summary}
+      Flood lights drown the stage as contestants file out from behind
+      the curtain\ldots
+
+      Queue: DJ Casper
+
+      As the music starts, the Lego Robot dance party begins!
+
+    \subsection{Objectives}
+    \begin{itemize}
+      \item Have fun getting the robot to dance!
+      \item (Intermediate) Get the robot to dance to the lyrics of the
+        Cha Cha Slide.
+      \item (Advanced) Get the robot to dance according to user button
+        presses.
+      \item (Cooperative) Work with the other students to design dance routines.
+
+    \end{itemize}
+    \subsection{Key Vocabulary Terms}
+    \begin{itemize}
+      \item Beat, Motor, Button
+      \item (Advanced) Synchronyze
+    \end{itemize}
+    \subsection{Materials}
+      \begin{itemize}
+        \item Any materials needed, including the standard set of
+          equipment
+        \item Standard ev3 brick
+        \item Laptops
+        \item Programming Cables
+        \item Smart phones with recordings of songs and bluetooth speaker
+        \item Demo Program: dancing to a pop song
+
+      \end{itemize}
+  \section{Procedure}
+    \subsection{Setup (5 minutes)}
+      Create a big space for the robots to dance!
+
+      Turn on computers and open the programming environment.
+      Build and turn on bots.
+    \subsection{Introduction (5 minutes)}
+      Introduce with a video of a musician that is recognizable to the
+      students. A video with a famous dance (like Thriller), might
+      be a fun way to start it.
```

Comment:
I like this. However, may be better to add more explicit directions than suggestions.

Reply:
Just to make it easier to follow. More of a curriculum than a guideline.